Update README.md (#1)
pypsa-meets-earth/.github: profile/README.md

## Hi there 👋,

> Welcome to the PyPSA meets Earth initiative, an independent global research and development initiative which aim is to provide cutting-edge energy modeling tool for any country in the world.

Reviewer: ```suggestion
Welcome to the PyPSA meets Earth initiative, an independent global research and development initiative which aims to provide cutting-edge energy modeling tool for any country in the world.
```

🌈 We are maintaining the following projects
- [PyPSA-Earth](https://github.com/pypsa-meets-earth/pypsa-earth) provides a way to build a high-resolution a power or sector-coupled models of any country, region or continent focusing at the transmission level
- [PyPSA-Distribution](https://github.com/pypsa-meets-earth/pypsa-distribution) is a tool for optimizing power or sector-coupled distribution systems with high spatial and temporal resolution
- [PyPSA-Earth-Documentation](https://github.com/pypsa-meets-earth/documentation) contains a collection of tutorial materials to get started with modeling and analyze the results
- [Earth-OSM](https://github.com/pypsa-meets-earth/earth-osm) is a package for extracting power infrastructure data from OpenStreetMap for any region of interest in a flexible and customizable way
- [PyPSA-Earth-Status](https://github.com/pypsa-meets-earth/pypsa-earth-status) intends to track the quality of PyPSA-Earth-based energy models by country, and coordinating the global validation efforts

> ❤️ PyPSA-Earth relays on [PyPSA ecosystem](https://github.com/pypsa) which provides tools to prepare data on renewable energy potentials, installed capacities, technologies costs as well as facilitate interfacing a model with a solver.

Reviewer: ```suggestion
❤️ PyPSA-Earth relies on [PyPSA ecosystem](https://github.com/pypsa) which provides tools to prepare data on renewable energy potentials, installed capacities, technologies costs as well as facilitate interfacing a model with a solver.
```

⭐ We are collaborating with following projects which can make modelers' life easier:

> - [Visualization toolkit](https://github.com/ben10dynartio/osm-power-grid-map-analysis) to quickly get insights inot OSM-extracted power grid datausing maps and graph analysis

Reviewer: Should we refer as Referenced initiatives?

Author: Sounds good!

> - [Visualization toolkit](https://github.com/ben10dynartio/osm-power-grid-map-analysis) to quickly get insights inot OSM-extracted power grid datausing maps and graph analysis

Reviewer: ```suggestion
- [Visualization toolkit](https://github.com/ben10dynartio/osm-power-grid-map-analysis) to quickly get insights into OSM-extracted power grid datausing maps and graph analysis
```

- [Energy-climate cookbook](https://github.com/matteodefelice/energy-climate-cookbook) provides an overview of educational resources relevant for energy modelers and gives a hand-on introduction into a using climate data in energy modeling

> 🙋‍♀️ Join our chat and voice exchange platform on [Discord](https://discord.gg/AnuJBk23FU) to get in contact with our global developers and users community

Reviewer: This is better placed before the linked the other initiatives

Author: Done :)
